sklearn/base.py

```diff
@@ -171,6 +181,26 @@ def _get_param_names(cls):
         # Extract and sort argument names excluding 'self'
         return sorted([p.name for p in parameters])
 
+    def get_url_link(self):
+        """Generating a link to the API documentation for given estimator."""
+        if self.__class__.__module__.split(".")[0] == self._doc_link_module:
```

Comment:
I think that it would make a more readable if we would rather start by this:
```suggestion
        if self.__class__.__module__.split(".")[0] != self._doc_link_module:
            # The document link is for the wrong module
            return ""
```
and the continue the function outside of an indented / if clause